tools/protoxform/protoxform.py

```diff
@@ -199,8 +199,12 @@ def CamelCase(s):
   google_imports = []
   infra_imports = []
   misc_imports = []
+  public_imports = []
```

Comment:
Should we be removing the public imports across versions? I.e. we have migrate.py do some fixup?

Reply:
That's not needed in this PR because the file with `import public` doesn't get upgraded (is empty), but potentially yes if we didn't move all messages to other file.